packages/office-ui-fabric-react/src/components/Toggle/Toggle.deprecated.test.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import * as renderer from 'react-test-renderer';
+import * as sinon from 'sinon';
+
+import { Toggle } from './Toggle';
+
+describe('Toggle', () => {
+  it('renders a label', () => {
+    const component = mount(<Toggle label="Label" />);
+    expect(
+      component
+        .find('.ms-Toggle-label')
+        .first()
+        .text()
+    ).toEqual('Label');
+  });
+
+  it('renders toggle correctly', () => {
+    const component = renderer.create(<Toggle label="Label" />);
```

Comment:
I think this file should only contain tests using deprecated props so.. not something like this for instance.

Reply:
Ah. Hmm. I was just looking at Button.deprecated.test.tsx to see what goes in such a test file, and found that it was virtually similar to Button.test.tsx.

Reply:
That's probably my fault for not cleaning it up when I deprecated Button props. I made a copy of it and then forgot to trim out tests that didn't test deprecated props.